sfputil/main.py

```diff
@@ -45,25 +47,6 @@
 MAX_LPL_FIRMWARE_BLOCK_SIZE = 116 #Bytes
 
 # TODO: We should share these maps and the formatting functions between sfputil and sfpshow
-QSFP_DATA_MAP = {
```

Comment:
this is still used by QSFP28/and QSFP+

Reply:
I did not remove it. I just move it to a common place. See sfp_helper.py.